### PR TITLE
Updated Parse and added CLI API to setup for Android

### PR DESCRIPTION
**Pull request by amsul**
This builds upon the fix for android app's crashing when not running (#28)

The change is, it allows to use the CLI to set the app id and client key:

```
cordova plugin add https://github.com/benjie/phonegap-parse-plugin --variable APP_ID=PARSE_APP_ID --variable CLIENT_KEY=PARSE_CLIENT_KEY
```

Which is then used to initialize Parse in the applications `onCreate` method:

```java
@Override
public void onCreate() {
    super.onCreate();
    ParsePlugin.initializeParseWithApplication(this);
}
```

To keep the JavaScript API consistent between platforms, `parsePlugin.initialize` still needs to be called to set up the default push callback.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ parsePlugin.initialize(appId, clientKey, function() {
 Usage
 -----
 ```
-<script type="text/javascript">
+<script type="text/javascript>
```
Comment (grrrian) [inline README.md]: why did you remove the "?

**Comment by grrrian** [comment]
clever stuff!

File: README.md (hunk 1 above)
Comment (amsul) [inline README.md]: I have no idea..definitely a mistake.

Comment (grrrian) [inline README.md]: Yea... I think you just need to merge with master. I like what you did though!

ian
